packages/init/src/utils/communication.ts

```diff
@@ -20,10 +20,13 @@ export async function getUserProfile(
   cookies: string,
   base = Utils.CONSTS.DEFAULT_BASE
 ): Promise<UserProfile> {
-  // note the auth server also provides a userId
+  const userServiceBase =
```

Comment:
note If the base was like `prismic.io | wroom.io`, it would make that easier to define and we could test it on other bases. 😄